UR5env_v0: #namespace
 
    #qlearn parameters
    
    alpha: 0.01 # Learning Rate
    alpha_decay: 0.01
    gamma: 1.0 # future rewards value 0 none 1 a lot
    epsilon: 1.0 # exploration, 0 none 1 a lot
    epsilon_decay: 0.995 # how we reduse the exploration
    epsilon_min: 0.01 # minimum value that epsilon can have
    batch_size: 128 # maximum size of the batches sampled from memory
    nepisodes: 1500
    nsteps: 1000
    #max_env_steps: None
    monitor: True
    quiet: False

    #desired_pose:
    #x: 0.0
    #y: 0.0
    #z: 1.0
     
    n_observations: 1 # Number of lasers to consider in the observations
    n_actions: 8 # Number of actions used by algorithm and task